libjlm/src/llvm2jlm/instruction.cpp

```diff
@@ -943,6 +975,7 @@ convert_instruction(
 	,	{llvm::Instruction::Or, convert_binary_operator}
 	,	{llvm::Instruction::Xor, convert_binary_operator}
 	,	{llvm::Instruction::Mul, convert_binary_operator}
+	,	{llvm::Instruction::FNeg, convert_unary_operator}
```

Comment:
Any reason why you added the fneg operation in the middle of all the binary operators?